test/service/test_manifest.py

```diff
@@ -82,6 +86,24 @@ def _get_manifest_url(self, format_, filters):
         service = ManifestService(StorageService())
         return service.get_manifest(format_, filters)
 
+
+class TestManifestEndpoints(ManifestTestCase):
+
+    def run(self, result: Optional[unittest.result.TestResult] = ...) -> Optional[unittest.result.TestResult]:
+        """
+        This patch introduces a context manager at the unittest level `run`. Allowing for
+        test in this suite to 'run' in the patched context.
+        The context manager patches 'ManifestService._can_use_cached_manifest' in order to
+        prevent a cached manifest from being used. It allows for testing of manifest
+        generation while ignoring cache functionality.
+        """
```

Comment:
```suggestion
        # Suppress caching of manifests to prevent false assertion positives
```

Generally speaking, this is not a comment for a docstring. It's also verbose and mostly obvious.